CONTRIBUTION.md

```diff
@@ -0,0 +1,103 @@
+- Crash log (If the crash log is known, although this is **NOT** required)
+    - If you have a long crash log, [Pastebin](https://pastebin.com/) can help (Make sure to set expiration to 'never')
+- Project file or database file in use (**NOT** required)
+
+A template from the GitHub issue will help you write a good bug/issue report.
+
+### Suggest enhancement
+Please check the following before suggesting an enhancement.
+
+- Search for existing similar issues
+
+And please follow the below to help members identify the suggestion.
+
+- Use a concise but descriptive title to identify the suggestion
+- Describe the detailed steps for the enhancement you want to propose
+- Attach a simple example of the proposal, already applied projects or screenshots
+
+### Translation
+Project translation can be done through the ['Qt Linguist'](http://doc.qt.io/qt-5/qtlinguist-index.html) program.
+Languages that are already being translated can be translated by modifying the corresponding translation file,
+and translations for new languages can be translated by adding a new entry to the TRANSLATION section of the src/src.pro file and then rebuilding the project.
+
+#### DO NOT translate key names in shortcuts
+Keyboard shortcuts get automatically localized by Qt, and translating key names may indeed break them. They show up on the translation file only for the rare occasions that one has to redirect a shortcut to another one in a specific language. But even then, you should use the English key names to get this working. Theoretically, one could follow Qt conventions for translated key names, and they should work, but the risk to stumble upon Qt bugs or moving conventions is high, so it is better to not translate keys and only remap the shortcut. If you want to preserve the original key sequence (recommended), just leave the translation empty and finished.
+
+## Styleguide
+Styleguides help keep the project's source code looking good and consistent.
+This can be of great help to existing and new contributors.
+
+### CPP source code
+#### Indent
+- 4 spaces(**NOT** tabs) for indentation
+
+#### Declare variables
+- Use lowerCamelCase
+- Declare only one variable per line
+- \* in pointer variable is attached to data type
+    - Ex) `int* ptr`
+- Avoid meaningless names (although these can be used for counter or temporary names)
+
+#### Whitespace
+- Do **NOT** use spaces after keywords
+- Leave a space after each comma
+
+#### Braces
+- Place braces on the line after the keyword
+- A brace can be omitted if the execution statement in the conditional/repeated statement consists of only one line
+
+### Git commit message
+Keeping Git commit messages consistent can help keep track of code history.
+
+- Limit the subject line to 72 character or less
+- Capitalize the subject line
+- Use the present tense
+- Use the imperative mood
+- Use the body to explain why
+- Reference related issue number
+    - Ex) `Add clearValue() method (#9999)`
```

Comment:
We usually don't [close issues from the commit](https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords), like a lot of other projects do, because we like to ask the submitter to download the nightly and confirm it is closed. Should we add a reference here?

Reply:
I meant to include issue number in commit to facilitate tracking between commits and issues, not to close the issue.
but if you think this doesn't help we can delete it. (because it's just my think 👀 )

Reply:
No, no, that's perfectly fine, but in addition, I propose to add a recomendation to not close the issue from the commit in order to request the user's confirmation. But that might too much detail, so it's open to discussion.

Reply:
I added the following recommendations for now.
>Do **NOT** close the issue from the commit(eg. Close #9999) before request the user's confirmation.

Reply:
Yeah, it's important we ask users to verify things before closing an issue.  Too many projects don't, and that just leads to incomplete fixes and unhappy users.